### ISSUE TITLE
Felicity wants retro items to be randomized so that she does not go through them in the order (and by author) as they were added

### ISSUE THREAD

**Issue by liamdarmody** [vmware-archive/postfacto#28]
As Felicity
I want retro items to be randomized when I use arrow keys to proceed through retro items
So that I do not go through them in the order (and by author) as they were added

---

**GIVEN** that I am facilitating a retro
**WHEN** I hit the arrow key to proceed to the next item
**THEN** the retro item that is presented should have been picked randomly and not based on a chronological order (which would often lead to items from the same author being read out one after the other)

Follow on from https://github.com/pivotal/postfacto/issues/26 (if required)

**Comment by crswty** [comment]
I suggest we hold off on doing anything on this one until we start seeing feedback from what we have. I think it might not be necessary and if it is, I think we should dig into if facilitators are genuinely going through in a random order or is there actually a heuristic there. Maybe reword it around not wanting to feel like we're going through person by person?

**Comment by Benjamintf1** [comment]
Why not in heart order?

**Comment by ryansorensen** [comment]
> Why not in heart order?

Agreed. We usually go in heart order.

**Comment by seadowg** [comment]
So @liamdarmody can probably provide a better answer to this but generally the idea about not sorting things via heart order in Postfacto is that retro facilitators shouldn't always just jump to the things that have the most +1s/votes/hearts - it's good for them to try and pace the discussion with a mix of positives/important items/rants etc.

Another reason is that if a team is using votes to "filter down" retro items to get through them each week then it's probably a sign that you need to be having more discussions outside your retro - not discussing items without votes is a great way to have "unpopular problems" never come up.

**Comment by liamdarmody** [comment]
Hi folks 👋

This question come up quite a lot and I understand why. I think it's worth giving some context as to why we believe there might be a better alternative to sorting retro items by heart order. (I've provided a similar response [here](https://github.com/pivotal/postfacto/issues/84#issuecomment-433490359).) 

The heart feature is supposed to mirror **+1s** in physical whiteboard retros. The great thing about **+1s** on the whiteboard is that the scale is extremely flexible. Some people might add a **+1** if they agree with the retro item. Others might add a **+100** if they **really agree** with the retro item. In any case, the ideal retro is one that gets through all the items on the board.

Some additional context can be found in these two Medium posts [here](https://builttoadapt.io/how-to-run-a-really-good-retrospective-8982bd839e16), and [here](https://medium.com/@nicolarushton/7-things-i-learned-about-retrospectives-while-building-postfacto-69863cc1ec41). I'll summarise the key points below:
- Retro is one practise that helps build a great team
- Talking about every item helps make sure everyone knows their voice is valued and equal
- Setting aside time each week purely for team communication and reflection builds a sense in the team that they can trust each other and that feedback is encouraged
- Getting through every retro item helps make sure everyone’s voice is heard
- These are the philosophies that have been driving product decisions for Postfacto

If you have any suggestions about how we can balance the above philosophies, whilst solving your team's needs, we'd love to hear them! 🙌 🇲🇱

**Comment by Benjamintf1** [comment]
Part of the idea is that I've seen postfacto sometimes used for larger contexts where it's not possible to talk about everything that wants to be talked about.
